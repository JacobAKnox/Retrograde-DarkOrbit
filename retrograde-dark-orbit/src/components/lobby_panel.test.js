import { render } from '@testing-library/react'
import LobbyPanel from './lobby_panel'

test('renders on-screen', () => {
  const page = render(<LobbyPanel/>);
  expect(page).toMatchSnapshot();
});
