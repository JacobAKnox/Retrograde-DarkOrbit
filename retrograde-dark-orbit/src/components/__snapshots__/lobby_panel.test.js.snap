// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders on-screen 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="border-4 w-full md:w-1/2 border-blue-400 min-h-[600px] bg-gray-900 md:mx-20 p-4 rounded-xl flex flex-row flex-wrap"
      >
        <div
          class="md:w-2/3 w-full"
        >
          <div>
            Mocked ChatBox
          </div>
        </div>
        <div
          class="flex flex-col w-full md:w-1/3 justify-end p-4 bg-gray-800 rounded-xl"
        >
          <div>
            Mocked UserList
          </div>
          <div>
            Mocked ReadyBox
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="border-4 w-full md:w-1/2 border-blue-400 min-h-[600px] bg-gray-900 md:mx-20 p-4 rounded-xl flex flex-row flex-wrap"
    >
      <div
        class="md:w-2/3 w-full"
      >
        <div>
          Mocked ChatBox
        </div>
      </div>
      <div
        class="flex flex-col w-full md:w-1/3 justify-end p-4 bg-gray-800 rounded-xl"
      >
        <div>
          Mocked UserList
        </div>
        <div>
          Mocked ReadyBox
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
